charts/brokencrystals/templates/deployment.yaml

```diff
@@ -200,8 +200,7 @@ spec:
             - name: CLUSTER
               value: "{{ .Values.cluster }}"
             - name: TIMEOUT
-              value: "{{ .Values.timeout | default "30000" }}"
-        restartPolicy: Always      
```

Comment:
i had to remove this line, and put it on bottom bellow sniffer container. This line is not container specific, it goes for all containers.